packages/blade/src/components/Icons/Icons.stories.tsx

```diff
@@ -79,3 +80,36 @@ export const Icon = IconTemplate.bind({});
 Icon.args = {
   icon: 'CreditCardIcon',
 };
+
+export const AllIcons: ComponentStory<ComponentType<IconProps>> = ({ ...args }) => {
+  return (
+    <Box>
+      {Object.keys(iconMap).map((icon, key) => {
+        const IconComponent = iconMap[icon];
+        return (
+          <Box
+            height={95}
+            width={125}
+            display="inline-flex"
+            flexDirection="column"
+            alignItems="center"
+            gap="spacing.6"
+            key={key}
+          >
+            <IconComponent {...args} />
+            <small
```

Comment:
I think the `small` tag here will crash the react native storybook. We can perhaps use `Box` here?